lavorare-insieme/dove-quando-e-come.md

```diff
@@ -0,0 +1,101 @@
+# Dove, quando e come
+
+La nostra settimana lavorativa si costituisce di 4 giorni dedicati a lavori per clienti esterni (non
+necessariamente sempre con lo stesso cliente) e di 1 giorno,
+[il venerdì](https://github.com/nebulab/playbook/blob/master/crescita-personale/venerdi.md), dedicato
+solo all'apprendimento, ai progetti interni o al contributo open source.
+
+Le giornate lavorative sono di 8 ore piuttosto flessibili. Per legge non si possono comunque
+superare le 13 ore giornaliere e le 48 ore settimanali. L'orario adottato dalla maggior parte
+di noi è compreso tra le 9:30 e le 18:30 considerando un'ora circa per la pausa pranzo.
+L'organizzazione del lavoro deve essere comunque vista nell'ottica del lavoro in team.
+Quindi chi sceglie di lavorare in orari diversi deve essere attento a non impedire ad un
+altro di svolgere correttamente il proprio lavoro.
+
+Riguardo al luogo la flessibilità è ancora maggiore. Nebulab al momento ha due sedi operative, una a
+Pescara e una a Latina, ma si è liberi di lavorare ovunque si voglia (volendo anche da casa
```

Comment:
`e una a` -> **ed** una a

Reply:
@matteogalliani http://www.accademiadellacrusca.it/it/lingua-italiana/consulenza-linguistica/domande-risposte/d-eufonica

Reply:
@aldesantis grazie!